css/quotes.css

```diff
@@ -1,193 +1,225 @@
-body {
-    margin: 0;
-}
-
-.form {
-    margin-left: 5%;
-    margin-right: 5%;
-    max-width: 90%;
-    font-family: 'Roboto', sans-serif;
-}
-
-.form__title {
-    font-size: 22px;
-    font-weight: bold;
-    text-align: justify;
-    margin-top: 50px;
-    margin-bottom: 10px;
-    color: rgba(0, 0, 0, 0.7)
-}
-
-.form__subtitles {
-    font-weight: bold;
-    text-align: justify;
-    margin-bottom: 18px;
-    color: rgba(0, 0, 0, 0.7)
-}
-
-.hr__quote {
-    border: 0;
-    margin-left: 0%;
-    margin-right: 50%;
-    width: 90px;
-    height: 1px;
-    background-color: #0086c3;
-}
-
-.hr__details {
-    margin-bottom: 36px;
-}
-
-.form__paragraph {
-    font-size: 12px;
-    font-weight: lighter;
-    margin-left: 5%;
-    margin-right: 5%;
-    text-align: justify;
-    margin-bottom: 40px;
-}
-
-
-/*
+                                                body {
+                                                    margin: 0;
+                                                }
+
+                                                .form {
+                                                    margin-left: 5%;
+                                                    margin-right: 5%;
+                                                    max-width: 90%;
+                                                    font-family: 'Roboto-Bold', sans-serif;
+                                                }
+
+                                                .form__title {
+                                                    font-size: 22px;
+                                                    text-align: justify;
+                                                    margin-top: 50px;
+                                                    margin-bottom: 10px;
+                                                    color: rgba(0, 0, 0, 0.7)
+                                                }
+
```

Comment:
I reckon you should erase all the empty lines from here to below

Reply:
I have a problem with the code format; i can't remove the empty spaces because when i save the changes, they appear again.

Reply:
ok, ps according to Juan it is not necessary that you erase them, better leave them that way.
Where do you make the changes in the format of your code is in visual code?

Reply:
In settings-save options but that doesn't work, either .editorConfig.

Reply:
You already tried to uninstall and reinstall

Reply:
Please follow BEm convections. for the save issue , close your work and reopen. later save again.